acro.json

```diff
@@ -2209,19 +2272,12 @@
         "acro": "FinARP", 
         "name": "Financial Agency Rotational Program"
     }, 
-    "CM": {
-        "note": "a cybersecurity term", 
-        "link_label": "", 
-        "link": "", 
-        "acro": "CM", 
-        "name": "configuration management"
-    }, 
-    "FFIEC": {
-        "note": "an interagency council", 
-        "link_label": "website", 
-        "link": "https://www.ffiec.gov/", 
-        "acro": "FFIEC", 
-        "name": "Federal Financial Institutions Examination Council"
+    "COPPA": {
+        "note": "requires parental consent when user testing involves children 13 and younger", 
+        "link_label": "web pageCOR", 
```

Comment:
arrrr. sorry, scott.
my alphabetized list was getting un-alphaed by jsonifying a python (unordered) dict. i switched to ordered dict, which will be sane now, but which replaces the whole shebang this time.